### PR TITLE
Civ 2824 create api to submit response

### PR DESCRIPTION
**Pull request by TatsyMaslova**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-2824

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by taomoh — DISMISSED** [review]
LGTM. I had only one question

File: src/main/java/uk/gov/hmcts/reform/civil/utils/CaseDataContentConverter.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import uk.gov.hmcts.reform.ccd.client.model.Event;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CaseDataContentConverter {
+
+    private CaseDataContentConverter(){
+
```
[the remark this answers is not in the document]
Comment (TatsyMaslova) [reply src/main/java/uk/gov/hmcts/reform/civil/utils/CaseDataContentConverter.java:13]: A very very good point Fergal!!! thank you!

File: src/main/java/uk/gov/hmcts/reform/civil/utils/CaseDataContentConverter.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    public static CaseDataContent caseDataContentFromStartEventResponse(
+        StartEventResponse startEventResponse, Map<String, Object> contentModified) {
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/reform/civil/utils/CaseDataContentConverter.java:17]: With public static methods, maybe add some Javadoc expressing this method does, as it'll be available to other users/projects?

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/dto/EventDto.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+
+import java.util.Map;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class EventDto {
+
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/dto/EventDto.java:16]: As I said on the client side, maybe this should be CaseEventDto, with EventDto as an interface?

**Review by osheaf01 — DISMISSED** [review]
Looks good, a couple of minor suggestions.

File: src/main/java/uk/gov/hmcts/reform/civil/utils/CaseDataContentConverter.java (hunk 2 above)
Comment (TatsyMaslova) [reply src/main/java/uk/gov/hmcts/reform/civil/utils/CaseDataContentConverter.java:16]: i think it is already states what it is doing and it is already used in the similar service that does events for case workers.

**Review by taomoh — APPROVED** [review]

**Review by laarmada — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/dto/EventDto.java (hunk 3 above)
Comment (osheaf01) [reply src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/dto/EventDto.java:16]: No, I know what DTOs are, I guess that I'm really saying is I'd prefer CaseEventDto as a name as it's more descriptive, given it's transferring a CaseEvent. But if all Events are CaseEvents, and no other sort of Event, then OK.

Comment (TatsyMaslova) [reply src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/dto/EventDto.java:16]: ah cool :) i thought you were proposing an interface for dto which is usually never done with DTO, so i thought perhaps you are not very familiar with it lol.

**Pull request merged**